### ISSUE TITLE
Update consent banner to meet accessibility and UX requirements

### ISSUE THREAD

**Issue by thekp** [BBC-archive/psammead#688]
~Blocked by: https://github.com/bbc/simorgh/issues/1934~

**Is your feature request related to a problem? Please describe.**
The Consent banner is live and no accessibility testing  has been documented for it.

After a quick review with Natalie and Catharine we have spotted some improvements that can be made.

- The consent banner contains a heading tag `<h2>`, therefore when a user is navigating headings via a screenreader, the first text read out will be from the consent banner rather than the article's header.
- The buttons are not clearly intractable and focus is not managed when it's activated

**Describe the solution you'd like**
- It should meet the screen reader UX - Change the `<h2>` element in the consent banner to a `<strong>` element
- Add text underline styling for all links and buttons when hovering/focusing
- Fix alignment for text 'Yes, I agree to cookies'
- Fix button so it can be seen when users change colour preferences in FF and when viewing in a custom high contrast mode on PC (It currently stays black in FF and I'm not sure it's right in HCM on PC either?)
- Confirm with UX if they want any additional styling for when users change colour preferences in FF and when viewing in high contrast mode on PC
- Move focus to correct place when the button is activated - see screen reader UX
- Check meets accessibility acceptance criteria
- Hold an accessibility swarm and go through this [checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by sareh** [comment]
The presence of the h2 and absence of focus/hover states was to ensure our Consent Banner was identical to that in Orbit, which is used for these banners on pages that are not in the Simorgh application. 
It would be good to confirm with @greenc05 & @nataliesmart that they're okay with there being this inconsistency in the markup/styling between different page types.

**Comment by greenc05** [comment]
Natalie and myself are ok with the inconsistency in the markup/styling between different page types.
